meteor/server/api/showStyles.ts

```diff
@@ -137,21 +167,47 @@ export async function removeShowStyleVariant(
 	await ShowStyleVariants.removeAsync(showStyleVariant._id)
 }
 
+export async function reorderAllShowStyleVariants(
+	context: MethodContext,
+	showStyleBaseId: ShowStyleBaseId,
+	orderedVariants: ShowStyleVariant[]
+): Promise<void> {
+	await assertShowStyleBaseAccess(context, showStyleBaseId)
+
```

Comment:
Minor thing: since you are just calling two methods here you don't need to have the extra space

Reply:
It has been removed!